README.md

```diff
@@ -30,18 +30,20 @@ Projects listed on this page are open source that Microsoft maintain or contribu
 |                                      | [KMSv2](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/3299-kms-v2-improvements) <br> ([blog](https://kubernetes.io/blog/2022/09/09/kms-v2-improvements/))                                                                                                   | Encryption at rest of Kubernetes data in etcd using Key Management Service (KMS) v2 API                                                                                                                                                                                       | API: v2alpha1                                                                                                                                                                                          | [GitHub issues](https://github.com/kubernetes/kubernetes/issues)                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                                               |
 |                                      | [Azure Workload Identity](https://github.com/Azure/azure-workload-identity) <br> ([releases](https://github.com/Azure/azure-workload-identity/releases))                                                                                                                           | Uses Kubernetes primitives to associate managed identities for Azure resources and identities in Azure Active Directory (AAD) with pods based on [Workload Identity federation](https://docs.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation) | Azure: incubation <br> API: N/A                                                                                                                                                                        | [GitHub issues](https://github.com/Azure/azure-workload-identity/issues)                                                                                                                                                                                                                                                                                                                       | [How to use](https://azure.github.io/azure-workload-identity/docs/installation.html)                                                                                                                                                                                                                                          |
```

Comment:
can you pls add a link for WI on AKS: https://learn.microsoft.com/en-us/azure/aks/workload-identity-overview

Reply:
@sozercan did you miss this?

Reply:
oh i missed this. thanks, updated!